.env.test

```diff
@@ -2,6 +2,7 @@
 NODE_ENV=production
 # Federation
 LOCAL_DOMAIN=cb6e6126.ngrok.io
+WEB_DOMAIN=cb6e6126.ngrok.io
```

Comment:
This was necessary because in my local development environment (`.env`) I have `WEB_DOMAIN` set to `localhost:3000` and this was causing the specs to fail for me.

We should arguably change these values to `mastodon.example` instead of a potentially valid web address.

Reply:
This turns out to fail because the end-to-end tests relied on the URLs generated to actually be accessible. I've gone with adjusting the end-to-end test workflow to also set `WEB_DOMAIN` not just `LOCAL_DOMAIN`

Reply:
> This was necessary because in my local development environment (`.env`) I have `WEB_DOMAIN` set to `localhost:3000` and this was causing the specs to fail for me.

Could you move your setting to `.env.development` so it doesnt interact w/ the test env?

> We should arguably change these values to `mastodon.example` instead of a potentially valid web address.

Agreed on that part!

Reply:
@mjankowski I can't, because .env.development is checked into git.

Reply:
@mjankowski I'm not sure if the dotenv loader we have supports `.env.development.local` files or not?

Reply:
It should, loads in this order: https://github.com/bkeepers/dotenv?tab=readme-ov-file#customizing-rails

Reply:
For now, I've removed the `WEB_DOMAIN` lines from this pull request.